New Checkout Drop-in v1.4.0-beta1 (#254)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/checkout/containers/overview.mdx

import OptionsTable from '@components/OptionsTable.astro';

> This section describes generic aspects common to all Checkout components that can be applied and used at the same way by all Checkout components that require them.

Reviewer: ```suggestion
In this section, we describe the common properties of Checkout components that can be applied and used in the same way.
```

};
```

> ## Title and header customization

Reviewer: ```suggestion
## Title and heading customization
```

> Each container provides a title and heading by default, but you may not want to display the title, or you may even want to provide a different customized title.

Reviewer: ```suggestion
Each container provides a title and a heading by default. If you want to hide the title or provide customized title, you can use the following interface.
```

> For this reason, an interface has been created to extend this functionality.

Reviewer: ```suggestion
```

### TitleProps interface

The `TitleProps` interface provides the following configuration options:

<OptionsTable
  compact
  options={[
    ['Option', 'Type', 'Req?', 'Description'],
    ['displayTitle', 'boolean', 'No', 'Displays the title of the container.'],
    ['slots', 'object', 'No', 'Object with the content to be displayed on the title.'],
  ]}
/>

> The containers receives an object as parameter which implements the `TitleProps` interface with the following properties:

Reviewer: ```suggestion
The container receives an object that implements the `TitleProps` interface with the following properties:
```

```ts
export interface TitleProps {
  displayTitle?: boolean;
  slots?: {
    Title?: SlotProps;
  };
}
```

> <li>The `displayTitle` property set to _true_ displays the title of the container.</li>

Reviewer: ```suggestion
<li>`displayTitle`: set to _false_ to display the title of the container.</li>
```

Author: If `displayTitle` is set to false, then the title will be hidden... I've just updated the property description to be more clear.

> <li>The `slots` property is an object containing the following properties:</li>

Reviewer: It looks like the properties are missing.

Author: The `slots` properties are described below.
